src/Geomap.js

```diff
@@ -58,8 +58,10 @@ export default class Geomap extends Viz {
 
     this._shape = constant("Circle");
     this._shapeConfig = assign(this._shapeConfig, {
+      ariaLabel: (d, i) => `${this._drawLabel(d, i)}, ${this._pointSize(d, i)}`,
       hoverOpacity: 1,
       Path: {
+        ariaLabel: (d, i) => `${this._drawLabel(d, i)}, ${this._colorScale(d, i)}.`,
```

Comment:
This needs a catch, similar to the one in d3plus-network regarding size, if the `colorScale` is not defined (only show the label in this case).